src/main/resources/config/application.yml

```diff
@@ -17,13 +17,17 @@ spring:
             indent_output: true
         serialization-inclusion: non_empty
     cloud:
-#        stream:
-#          schemaRegistryClient:
-#            endpoint: http://localhost:8990
-          bindings:
-            output:
-              destination: avro-events
-              contentType: 'application/json'
+        stream:
+            bindings:
+                input:
+                    destination: avro
+                output:
+                    destination: avro
+    rabbitmq:
```

Comment:
Yes. For now this part just provides the default configuration but this is the place where you can configure the rabbitmq connection for spring cloud stream.